packages/yuudachi/src/functions/reports/getReport.ts

```diff
@@ -18,3 +20,19 @@ export async function getReport(guildId: string, reportId: number) {
 
 	return transformReport(rawReport);
 }
+
+export async function getPendingReportsByTarget(guildId: string, targetId: string): Promise<Report> {
```

Comment:
```suggestion
export async function getPendingReportByTarget(guildId: string, targetId: string): Promise<Report> {
```

Reply:
Needs to be changed everywhere its used too.

Reply:
Fixed in 134fd8c